README.rst

```diff
@@ -131,3 +131,16 @@ Perform automated testing with the tox tool:
 
     $ tox
 
+
+Installing & enabling pre-commit
```

Comment:
I think it should be part of **Preparing the environment** 

https://github.com/vmware/repository-service-tuf-cli/blob/e0cae3379ef9ee618f3be8ee0934d10a03ec047b/README.rst#preparing-the-environment